src/Denner/Client/ArticlesClient.php

```diff
@@ -17,6 +18,28 @@
 class ArticlesClient extends DennerClient
 {
     /**
+     * @param array $params
+     * @param string $broadcastAction
+     * @return Response\ResourceResponse
+     */
+    public function fetchAdvertisedArticle(array $params = array(), $broadcastAction = null)
```

Comment:
On alternative could implement the use of the `$broadcastAction` as second parameter in the parent __call, if `$args[1]` is set.